src/Angor/Client/Pages/View.razor

```diff
@@ -443,60 +451,70 @@
-
-        project = storage.GetInvestmentProjects().FirstOrDefault(p => p.ProjectInfo.ProjectIdentifier == ProjectId);
+        projectStats.Loading = true;
+        findInProgress = false;
+        error = string.Empty;
 
-        if (project is InvestorProject findProject)
-        {
-            invested = findProject.InvestedInProject(); // TODO: need to scan for the invested projects when opening on a new browser
-            myProjectExplorerLink = _NetworkConfiguration.GetExplorerUrl().Url + $"/transaction/{findProject.TransactionId}";
-        }
-        else
+        try
         {
-            project = storage.GetFounderProjects().FirstOrDefault(p => p.ProjectInfo.ProjectIdentifier == ProjectId);
+            NostrClients = _NetworkConfiguration.GetDefaultRelayUrls().Select(_ => _.Url.ToString()).ToList();
 
+            project = storage.GetFounderProjects().FirstOrDefault(p => p.ProjectInfo.ProjectIdentifier == ProjectId);
             if (project != null)
             {
                 founder = true;
+                projectStats.Loading = false;
+                SetProjectLinksAndRefreshBalance();
+                return;
             }
-            else
+
+            project = storage.GetInvestmentProjects().FirstOrDefault(p => p.ProjectInfo.ProjectIdentifier == ProjectId);
+            if (project is InvestorProject investorProject)
             {
-                project = SessionStorage.GetProjectById(ProjectId);
+                invested = investorProject.InvestedInProject();
+                myProjectExplorerLink = _NetworkConfiguration.GetExplorerUrl().Url + $"/transaction/{investorProject.TransactionId}";
+                projectStats.Loading = false;
+                return;
+            }
 
-                if (project == null)
-                {
-                    findInProgress = true;
+            project = SessionStorage.GetProjectById(ProjectId);
+            if (project != null)
+            {
+                await SetProjectLinksAndRefreshBalance();
+                projectStats.Loading = false;
+                StateHasChanged();
+                return;
+            }
 
-                    var projectIndexerData = await _IndexerService.GetProjectByIdAsync(ProjectId);
+            findInProgress = true;
+            var projectIndexerData = await _IndexerService.GetProjectByIdAsync(ProjectId);
 
-                    if (projectIndexerData != null)
-                    {
-                        project = new Project { CreationTransactionId = projectIndexerData.TrxId };
-                       
-                        _RelayService.LookupProjectsInfoByEventIds<NostrEvent>(x =>
+            if (projectIndexerData != null)
+            {
+                project = new Project { CreationTransactionId = projectIndexerData.TrxId };
+
+                _RelayService.LookupProjectsInfoByEventIds<NostrEvent>(x =>
```

Comment:
@dangershony @DavidGershony on this relay call i just took what's in main and relaying on it